source/faq.txt

```diff
@@ -20,20 +20,18 @@ Support.
 Can I perform reads or writes to my destination cluster while ``mongosync`` is syncing?
 ---------------------------------------------------------------------------------------
 
-You can perform reads during synchronization if ``canCommit`` is ``true``. 
-However, the data that you read is 
-:term:`eventually consistent <eventual consistency>`, meaning that you might not 
-always read the latest writes.
+You can perform reads during synchronization at any time. However, the data 
+that you read is :term:`eventually consistent <eventual consistency>`, meaning 
+that you might not always read the latest writes. For up-to-date reads, wait 
+for the migration to commit. To learn more, see :ref:`mongosync-considerations`.
 
-During an ongoing sync, you can write to any non-synced namespaces in the 
-destination cluster as long as the source cluster doesn’t include a namespace 
-with the same name. If you write to a synced namespace before issuing a 
-:ref:`commit <c2c-api-commit>` and while ``canWrite`` is ``false``, the behavior 
-is undefined. To avoid this undefined behavior, you can enable 
-:ref:`write blocking <c2c-dr-write-blocking>`.
+If you write to a synced namespace before issuing a :ref:`commit 
+<c2c-api-commit>` and while ``canWrite`` is ``false``, the behavior is 
+undefined. To avoid this undefined behavior, you can enable :ref:`write blocking 
```

Comment:
I think we should say to ensure that no writes happen to synced namespaces instead of "to avoid this"